### DISCUSSION TITLE
Welcome to CustomWebView Discussions!

### DISCUSSION THREAD

**Discussion by vknow360** [vknow360/CustomWebView#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tomislav-Tomsic** [comment]
Hi all, and best of everything in New Year. As for short introduction, for  while now I am interested in expanding MIT App Inventor2. Ever since I was teaching IT/Software development in schools here in Zagreb, Croatia, EU. MIA2 is a great tool, not just for teaching. Currently I am mostly interested in visible components, which lead me to CustomWebView extension, predictable isn't it :)

**Reply by vknow360** [reply]
Hi @Tomislav-Tomsic 
Glad to know that.
Hope the extension could be useful to you.

**Comment by iamwsumit** [comment]
hello sunny please also upload your Date Tool extension source code please

**Reply by vknow360** [reply]
Hi @Sumit1334 
Sorry, I can't.

**Reply by iamwsumit** [reply]
Ok @vknow360 
 please
 just help me to add online time get blocks in my extension

**Reply by vknow360** [reply]
@Sumit1334 you will have to use an NTP server.
See here how:
https://stackoverflow.com/questions/31129835/how-to-get-time-from-network-in-android

**Comment by iamwsumit** [comment]
I want to get time from an api worldtimeapi.org

**Reply by vknow360** [reply]
That is not the recommended method.

**Comment by k3nzaki** [comment]
Hello, I am currently working with an APP that uses the extension. When i try to click on a link which opens a new tab. It wont work on the customwebview? Can anyone help me on how to get the target link so that i can set the new screen window href with it. Thank you.

**Reply by vknow360** [reply]
Hi, you will have to handle new tab requests correctly.
https://sunnythedeveloper.in/snippets-related-to-customwebview-extension/#Handling_New_Tab_Request_and_Back_Press

You can skip the backpress part if you want.

However, it is not possible to get target url without loading it into a new webview.
